src/pages/product.html

```diff
@@ -0,0 +1,227 @@
+        </div>
+      </div>
+      <div class="feature-card gap-4 w-[378px]">
+        <img class="feature-card__title-icon" src="../assets/icons/protected.svg" alt="security icon">
+        <div class="section-text">
+          <h3 class="headline">Easy notes organization
+          </h3>
+          <p class="description">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy.</p>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="halved-section halved-section--pushed-down">
+    <div class="section-text">
+      <h4 class="section-text__label">Lead Generation</h4>
+      <h2 class="section-text__heading">Data-driven client feedback</h2>
+      <p class="description desciption--why-us-section">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+        diam nonumy eirmod tempor invidunt ut labore et dolore
+        magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
+        gubergren.</p>
+    </div>
+    <img class="w-[466.98px]" src="../assets/images/impressions_product.png" alt="">
+  </section>
+  <section class="halved-section halved-section--pushed-down">
+    <img class="w-[466.98px]" src="../assets/images/impressions_product.png" alt="">
+    <div class="section-text">
+      <h4 class="section-text__label">Time tracker</h4>
+      <h2 class="section-text__heading">Track your project performance</h2>
+      <p class="description desciption--why-us-section">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+        diam
+        nonumy eirmod tempor invidunt ut labore et dolore
+        magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
+        gubergren.</p>
+    </div>
+  </section>
+  <section class="halved-section halved-section--pushed-down">
+    <div class="section-text">
+      <h4 class="section-text__label">Lead Generation</h4>
+      <h2 class="section-text__heading">Data-driven client feedback</h2>
+      <p class="description desciption--why-us-section">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
+        diam nonumy eirmod tempor invidunt ut labore et dolore
+        magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd
+        gubergren.</p>
+    </div>
+    <img class="w-[466.98px]" src="../assets/images/Image.png" alt="">
+  </section>
+  <section class="halved-section halved-section--pushed-down">
+    <img class="w-[466.98px]" src="../assets/images/impressions_product.png" alt="">
+    <div class="section-text">
+      <h4 class="section-text__label">Time tracker</h4>
+      <h2 class="section-text__heading">Track your project performance</h2>
+      <p class="description desciption--why-us-section">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed
```

Comment:
`desciption--why-us-section` is included without it's default Element (desciption).